modules/nw-ingress-sharding.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc
+// * ingress-operator.adoc
+
+[id="nw-ingress-sharding_{context}"]
+= Configuring ingress controller sharding
+
+Ingress Controllers that have permission to view all of the labels in all
+projects can select routes to admit based on the labels. This is called Ingress
```

Comment:
The first sentence doesn't make sense: (1) the ingress controller needs to be able to list certain resources, not just their labels (whatever that even means); (2) the resources that an ingress controller needs to have permission to list are namespaces, services, endpoints, and routes; and (3) any ingress controller that the ingress operator creates has permissions to list all such resources in the cluster, so it is unnecessary and probably confusing to begin by talking about permissions.